### PR TITLE
feat: adding bag icon to navbar

### PR DESCRIPTION
**Pull request by JackShort**
* adds shopping bag icon to navbar
* fix: changes size of ellipsis icon in menu bar

### CODE REVIEW

File: src/components/NavBar/Navbar.tsx [hunk 1]
```diff
@@ -64,6 +64,9 @@ const PageTabs = () => {
 }
 
 const Navbar = () => {
+  const { pathname } = useLocation()
+  const isNftPage = pathname.startsWith('/nfts')
```
Comment (cbachmeier) [inline src/components/NavBar/Navbar.tsx:68]: Can you rebase this PR after merging the nfts fix to verify that the TagIcon shows as expected for Sell

File: src/components/NavBar/ShoppingBag.css.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { style } from '@vanilla-extract/css'
+import { sprinkles } from 'nft/css/sprinkles.css'
+
+export const bagQuantity = style([
+  sprinkles({
+    position: 'absolute',
+    top: '4',
+    right: '4',
+    backgroundColor: 'blue400',
```
Comment (cbachmeier) [inline src/components/NavBar/ShoppingBag.css.ts:9]: is this blue for dark and lightmode or do we use pink in lightmode?

**Review by cbachmeier — COMMENTED** [review]
Small question and ping me for a re-review after rebasing the nft/nfts fix so I can verify shopping tag for list

File: src/components/NavBar/Navbar.tsx (hunk 1 above)
Comment (cbachmeier) [reply src/components/NavBar/Navbar.tsx:68]: I merged latest locally and was seeing the shopping bag, not the tag for sell

Comment (JackShort) [reply src/components/NavBar/Navbar.tsx:68]: yes

File: src/components/NavBar/ShoppingBag.css.ts (hunk 2 above)
Comment (JackShort) [reply src/components/NavBar/ShoppingBag.css.ts:9]: will check in with design

**Pull request merged**